crates/anthropic/src/anthropic.rs

```diff
@@ -90,71 +90,91 @@ impl Model {
     }
 
     pub fn from_id(id: &str) -> Result<Self> {
+        if id.starts_with("claude-opus-4-thinking") {
```

Comment:
Writing this in the early-return style makes it easier to add/remove cases and produces nicer diffs.